.gitignore

```diff
@@ -7,7 +7,6 @@ __pycache__/
 
 *.pyc
 .idea
-*.json
```

Comment:
Seeding data is best done programmatically. This is also a separate issue: https://github.com/openoakland/TrashTalk/issues/81

I've added a commanded to programmatically seed data into the database, so check it out and see if we need to add more data/models there. 

That is also a separate PR and shouldn't be included in this one.

Reply:
Why is it best done programatically? Wouldn't it be best to just use migrations or fixtures, since these are the Django recommended ways? https://docs.djangoproject.com/en/2.0/howto/initial-data/

Reply:
Because fixtures require unnecessary maintenance.  Running a command is simpler and stays current with models.